imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -20,10 +20,6 @@ default_attrs: &default
     VALIDMAX: 9223372036854775807
     VAR_TYPE: data
```

Comment:
You can add `DEPEND_0` here and then you don't need to add it in `codice_l1a.py`.